api/envoy/extensions/filters/http/json_to_metadata/v3/json_to_metadata.proto

```diff
@@ -108,6 +110,10 @@ message JsonToMetadata {
     // Allowed empty content-type for json to metadata transformation.
     // Default to false.
     bool allow_empty_content_type = 3;
+
+    // Allowed content-type by regex match for json to metadata transformation.
+    // This can be used in parallel with allow_content_types.
+    repeated type.matcher.v3.RegexMatcher allow_content_types_regex = 4;
```

Comment:
Probably should just be a single, since regex can express conjunction, i.e.

```suggestion
    type.matcher.v3.RegexMatcher allow_content_types_regex = 4;
```

Reply:
> Probably should just be a single, since regex can express conjunction, i.e.

updated to single